packages/fedify/src/federation/callback.ts

```diff
@@ -15,6 +16,17 @@ export type NodeInfoDispatcher<TContextData> = (
   context: RequestContext<TContextData>,
 ) => NodeInfo | Promise<NodeInfo>;
 
+/**
+ * A callback that dispatches a array of {@link Link}.
+ *
+ * @template TContextData The context data to pass to the {@link Context}.
+ * @param resource The URL that provide additional info to dispatcher
```

Comment:
I think this description is inappropriate:

```suggestion
 * @param resource The URL queried via WebFinger.
 * @returns Links related to the queried resource. 
```